index.html

```diff
@@ -68,7 +68,7 @@ <h3>We start from here ? </h3>
               />                
             </section>
             <section>
-                <h3>No! We start from here </h3>
+                <h3>No! We start from here</h3>
```

Comment:
Maybe it better to say
```suggestion
                <h3>No! We should start from here</h3>
```
?